ner_v2/detectors/numeral/number/README.md

```diff
@@ -109,12 +109,12 @@ Below is the brief about how to create data files `numerals_constant.csv`, `unit
 
 2. **units.csv**:  This files contains the vocabs of units for number and their correspoding all variants possible .
 
-   | unit_value |                        unit_variants                         |
-   | :--------: | :----------------------------------------------------------: |
-   |   rupees   | rupees \| rupay \| paisa \| paise \| inr \| रूपीस \| रुपया \| रूपए\| पैसा\| पैसे\| ₹ |
-   |   dollar   |                  Dollar \| usd \| डॉलर \| $                  |
+   | unit_type | unit_value |                        unit_variants                         |
+   | --------- | :--------: | :----------------------------------------------------------: |
+   | currency  |   rupees   | rupees \| rupay \| paisa \| paise \| inr \| रूपीस \| रुपया \| रूपए\| पैसा\| पैसे\| ₹ |
+   | currency  |   dollar   |                  Dollar \| usd \| डॉलर \| $                  |
 
-   Here, 1st column will contain the value of unit which will be return by detector module, while 2nd column contain all the possible variants of units in language and english script.
+   Here, 1st column have the type of unit variants it have correspoding to unit variants, 2nd contain the value of unit which will be return by number detector and 3rd column contains all the possible variants of units in language and english script.
```

Comment:
Phrasing:
Here, the 1st column contains the type of unit (E.g. dollars, euros are "currency",  centimetre, metre, kilometre are "distance"), 2nd column contains the value of unit which will be returned by number detector and 3rd column contains all the possible variants of that unit value (delimited by pipe  `|`) for the language you are adding. (It is recommended to add Romanised versions of the variants you are adding)